recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "tqdm" %}
-{% set version = "4.8.4" %}
+{% set version = "4.9.0" %}
```

Comment:
I know (I released it) but wanted to make sure v4.9 was also up on conda in case people wanted it. will bump again to 4.10 (and an imminent 4.11) after this PR